lib/l10n/app_fr.arb

```
@@ -289,15 +289,15 @@
   "currency_settings_title": "Sélection de devises",
   "currency_settings_subtitle": "Sélectionnez vos devises préférées",
   "available_currencies": "Devises disponibles",
-  "selected_currencies": "Devises sélectionnées", 
+  "selected_currencies": "Devises sélectionnées",
   "no_currencies_available": "Aucune devise disponible depuis le serveur",
   "select_currencies_hint": "Sélectionnez des devises dans la liste ci-dessus",
   "preview_title": "Aperçu",
   "tap_to_cycle": "Appuyez pour parcourir les devises",
   "calculating_text": "Calcul...",
```

RuboCop (1.86.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)